amaechi-chuks/iReporter
# Ch code structure #162383839

**What does this PR do?**
This PR structure the codebase for easy interpretation and decode

**Description of tasks**

Ensure proper indentation
Remove all dead codes
Ensure the use of a proper variable names

**How can this PR be tested**

Using any Code editor of your choice
Open the codebase and check for the following:
- Proper indentation
- variable naming convention
- Zero dead codes

**Link to relevant story**
#162383839
